.harness/Pipeline.yaml

```diff
@@ -0,0 +1,55 @@
+pipeline:
+  name: test-signal-public-release
+  identifier: testsignalpublicrelease
+  projectIdentifier: DhrubaCI
```

Comment:
Remove the mentions *dhruv* and use a more generic name

Reply:
fixed